rct229/data_fns/table_3_5_3_fns.py

```diff
@@ -0,0 +1,75 @@
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+
+# This dictionary maps the ChillerCompressorOptions enumerations to
+# the corresponding chiller types
+# ashrae_90_1_table_3_5_3.json
+water_chiller_compressor_type_map = {
+    "SCREW": "displacement",
+    "CENTRIFUGAL": "centrifugal",
+    "RECIPROCATING": "displacement",
+    "SCROLL": "displacement",
+    "POSITIVE_DISPLACEMENT": "displacement",
+    "SINGLE_EFFECT_INDIRECT_FIRED_ABSORPTION": "displacement",
+    "DOUBLE_EFFECT_INDIRECT_FIRED_ABSORPTION": "displacement",
+    "SINGLE_EFFECT_DIRECT_FIRED_ABSORPTION": "displacement",
+    "DOUBLE_EFFECT_DIRECT_FIRED_ABSORPTION": "displacement",
+    "OTHER": "displacement",
+}
+
+# minimum capacity and maximum capacity in the ashrae_90_1_table_3_5_3
+minimum_capacity_threshold_list = [0, 150, 300]
+maximum_capacity_threshold_list = [9999.99, 299.99, 149.99]
+
+
+def table_3_5_3_lookup(compressor_type, capacity):
+    """Returns the chiller data based on compressor type and capacity
+    required by ASHRAE 90.1 Table 3.2
+    Parameters
+    ----------
+    compressor_type : str
+        One of the ChillerCompressorOptions enumeration values
+    capacity: quantity
+        chiller rated capacity in Tons
+    Returns
+    -------
+    dict
+        { minimum_full_load_efficiency: Quantity - the minimum full load efficiency
+          minimum_party_load_efficiency: Quantity - the minimum part load efficiency
+        }
+    """
+    compressor_category = water_chiller_compressor_type_map[compressor_type]
+
+    minimum_capacity = 0
+    for min_threshold in minimum_capacity_threshold_list:
+        if capacity > min_threshold:
+            minimum_capacity = min_threshold
+
+    maximum_capacity = maximum_capacity_threshold_list[-1]
```

Comment:
I think you need `maximum_capacity = maximum_capacity_threshold_list[0]`. Otherwise, the `maximum_capacity` corresponding to a capacity of 1000000 would be 149.99.